roles/confluent.schema_registry/tasks/main.yml

```diff
@@ -19,13 +19,17 @@
     name: "{{item}}{{confluent_package_redhat_suffix}}"
     state: latest
   loop: "{{schema_registry_packages}}"
-  when: ansible_os_family == "RedHat"
+  when:
+    - ansible_os_family == "RedHat"
+    - not archive_file_enabled|bool
```

Comment:
Im leaning towards having a variable `installation_method` which defaults to `package`, but can also be `archive`
wdyt?

Reply:
I can change that variable name. Look for a new push in a bit.

Reply:
Fixed in commit aa7ff965